packages/cli/internal/pkg/cli/actionable/error.go

```diff
@@ -0,0 +1,38 @@
+package actionable
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Error struct {
+	Cause           error
+	SuggestedAction string
+}
+
+func (e *Error) Error() string {
+	return fmt.Sprintf("an error occurred caused by: %s\nsuggestion: %s\n", e.Cause, e.SuggestedAction)
+}
+
+func NewError(cause error, suggestedAction string) error {
+	err := new(Error)
+	err.SuggestedAction = suggestedAction
+	err.Cause = cause
+
+	return err
+}
+
+func FindSuggestionForError(cause error, errorToSuggestionMap map[string]string) error {
+	err := new(Error)
```

Comment:
This is unnecessery allocation. In case the error message doesn't match, this allocated object will be waster. Please move it inside "if" statement and use struct literal syntax.

For more struct creation examples please see:
- https://gobyexample.com/structs